### PR TITLE
Add big_query_query_to_table_it to python SDK

### PR DESCRIPTION
**Pull request by boyuanzz**
Add big_query_query_to_table_it to python SDK, include test cases: legacy query, standard query and new types.

### CODE REVIEW

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

File: sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+from apache_beam.io.gcp.tests.bigquery_matcher import BigqueryMatcher
+from apache_beam.testing.pipeline_verifiers import PipelineStateMatcher
+from apache_beam.testing.test_pipeline import TestPipeline
+
+BIG_QUERY_DATASET_ID = 'python_query_to_table_'
+NEW_TYPES_INPUT_TABLE = 'python_new_types_table'
+NEW_TYPES_OUTPUT_SCHEMA = (
+  '{"fields": [{"name": "bytes","type": "BYTES"},'
+  '{"name": "date","type": "DATE"},{"name": "time","type": "TIME"}]}')
+NEW_TYPES_OUTPUT_VERIFY_QUERY = ('SELECT date FROM [%s];')
```
Comment (boyuanzz) [inline sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py]: Ideally, this verify query should also query bytes and time. But there are some bugs within current bigquery client version we are using. For time, it cannot process .9999999: https://github.com/GoogleCloudPlatform/google-cloud-python/issues/5750. For bytes, it cannot query bytes type properly.

**Comment by boyuanzz** [comment]
R: @pabloem 
cc: @aaltay

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
Run Python PostCommit

**Comment by boyuanzz** [comment]
There are some other failures in PostCommit, which aren't related to my changes, eg out of quotas: https://scans.gradle.com/s/fias56tanijtc/console-log#L2358
IT succeeded with dataflow runner: https://scans.gradle.com/s/fias56tanijtc/console-log#L2353
IT succeeded with direct runner: https://scans.gradle.com/s/fias56tanijtc/console-log#L243

**Comment by pabloem** [comment]
This is excellent. Thank you Boyuan.

**Pull request merged**